### PR TITLE
Update Vaxrank to 0.3.0 and its arguments/defaults

### PR DESCRIPTION
**Pull request by ihodes**
fixes https://github.com/hammerlab/biokepi/issues/367

### CODE REVIEW

**Comment by smondet** [comment]
LGTM, just one question: are 2.x versions now totally irrelevant/useless?
(if not we could have kept the code arround and manage both versions)

**Comment by ihodes** [comment]
Good point, but yeah, those old versions are completely superseded.

**Comment by ihodes** [comment]
Waiting on changes per https://github.com/hammerlab/epidisco/pull/105#issuecomment-265286924

**Comment by ihodes** [comment]
Okay, this is ready for final review!

File: src/bfx_tools/vaxrank.ml [hunk 1]
```diff
@@ -44,39 +34,60 @@ module Configuration = struct
...
     =
-    `Assoc [
+    `Assoc ([
       "name", `String name;
       "vaccine_peptide_length", `Int vaccine_peptide_length;
       "padding_around_mutation", `Int padding_around_mutation;
       "max_vaccine_peptides_per_mutation", `Int max_vaccine_peptides_per_mutation;
       "max_mutations_in_report", `Int max_mutations_in_report;
       "min_mapping_quality", `Int min_mapping_quality;
       "min_reads_supporting_variant_sequence",
```
Comment (julia326) [inline src/bfx_tools/vaxrank.ml]: worth updating this string also for consistency

Comment (ihodes) [inline src/bfx_tools/vaxrank.ml]: Nice catch! Fixed.

**Comment by ihodes** [comment]
@smondet this is ready for merging—confirmed that it works.

**Pull request merged**